README.md

```diff
@@ -637,6 +638,31 @@ SwiftyStoreKit.verifyReceipt(using: appleValidator) { result in
     }
 }
 ```
+#### Get distinct purchase identifiers 
+
+You can retrieve all product identifiers with the `getDistinctPurchaseIds` method:
+
+```swift
+let appleValidator = AppleReceiptValidator(service: .production, sharedSecret: "your-shared-secret")
```

Comment:
Let's leave some whitespaces between lines. Maybe it would be better.

Reply:
I have not left any white spaces to ensure consistency with the aforewritten parts.